features/no_property_address.feature

```diff
@@ -0,0 +1,14 @@
+@us85a @add_property_address
+  Feature: Conveyancer provides a single property address
+  As a conveyancer I want to be able to provide a property address
+  So that it can be included on the mortgage deed
+
+
+Acceptance Criteria
+  Property provided must be a single property address
+
+@us85a
+Scenario: No property address provided
+  Given I do not have a property address to provide
+  When I create the deed via the Deed API
+  Then a status code of "400" is returned
```

Comment:
Agree with what you mentioned that a blank property address should come back with a schema validation error with a sechema error message so would add a step to check this message is returned when they sort that out.